tfhe/src/integer/server_key/radix/tests.rs

```diff
@@ -117,6 +118,10 @@ create_parametrized_test_classical_params!(integer_default_scalar_slice);
 create_parametrized_test_classical_params!(integer_default_scalar_slice_assign);
 create_parametrized_test_classical_params!(integer_smart_scalar_slice);
 create_parametrized_test_classical_params!(integer_smart_scalar_slice_assign);
+create_parametrized_test!(integer_unchecked_min {
+    coverage => {COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS},
+    no_coverage => {PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64}
+});
```

Comment:
```suggestion
create_parametrized_test_classical_params!(integer_unchecked_min);
```

Reply:
This panics for 1_MESSAGE_1_CARRY with the following message: `At least 4 bits of space (message + carry) are required to be able to do comparisons.` So I skipped that one.